### ISSUE TITLE
Updating `eventsList` not possible anymore

### ISSUE THREAD

**Issue by rwbr** [rwbr/flutter_neat_and_clean_calendar#83]
When rendering the widget with a changed `eventsList` the new set of events will have no influence on the view. The internal data of the widget can set only once during the initial creation. 

It must be possible to update the contents of the calendar through re-rendering the widget.

**Comment by m-gug** [comment]
I still have no correct behavior when I update the events after the initial loading. I have had this problem since 4.0.
I have the calendar in the expanded view and initially load all events for the currently visible month. When I switch to a previous month, the events for the now visible month are reloaded and added via onDateSelected. However, the new entries do not appear in the view.
However, if I close the expanded view and then switch back to the previous month week by week, the entries are displayed correctly in the week view.
Should I create a new issue?
